packages/expo-video/src/VideoView.types.ts

```diff
@@ -113,4 +113,12 @@ export interface VideoViewProps extends ViewProps {
    * A callback to call after the video player exits fullscreen mode.
    */
   onFullscreenExit?: () => void;
+
+  /**
+   * A callback to call after the mounted `VideoPlayer` has rendered the first frame into the `VideoView`.
+   * This event can be used to hide any cover images used to hide the initial loading of the player.
```

Comment:
```suggestion
   * This event can be used to hide any cover images that conceal the initial loading of the player.
```

A tiny suggestion: let's not repeat the same verb to avoid confusion.